tests/e2e/send.test.js

```diff
@@ -149,6 +149,7 @@ async function sendToken(page) {
 
   const sendButton = await page.getByTestId('send-button', true);
   await sendButton.click();
+  await page.waitForTimeout(15000);
```

Comment:
Let's try to do something... would you like to replace `await page.waitForTimeout(15000` to `page.waitForNavigation()`